airflow/datasets/manager.py

```diff
@@ -68,6 +68,9 @@ def _queue_dagruns(self, dataset: DatasetModel, session: Session) -> None:
         for dag_id in consuming_dag_ids:
             session.merge(DatasetDagRunQueue(dataset_id=dataset.id, target_dag_id=dag_id))
 
+    def resolve_dataset_model(self, dataset: Dataset) -> DatasetModel:
+        return DatasetModel(uri=dataset.uri, extra=dataset.extra)
```

Comment:
Since this is a public method, consider adding `**kwargs` for forward-compatibility. That way, if the signature changes in the future, plugin authors don't have to resort to hacks (like figuring out Airflow version or checking the function signature) before calling it.

Reply:
@blag I don't think that's necessary here, any "kwargs" should come from the dataset object passed in, otherwise there is no way for the user to get them in to this call in the first place.